main.scss

```diff
@@ -3,7 +3,66 @@
 @import "o-typography/main";
 @import "o-icons/main";
 
-@import 'src/scss/variables';
+@import "src/scss/variables";
 @import "src/scss/color-use-cases";
+
+// ensure that the placeholders are only output once
+$_o-quote-placeholders-defined: false !default;
 @import "src/scss/base";
```

Comment:
Hmm I worry wrapping imports in variables might be prone to refactor whoopsies. It's not clear that those files should contain placeholders only. If I remember right Bren tried to bring placeholders back before, until n-ui css splitting ruined the fun. Their approach was to have a global list of placeholders which had been output and perform the check within the mixin which does the extending. I think it was a good approach as it keeps the placeholder and the control for outputting that placeholder once in the same place. 

I think it was something like:
```scss
@mixin oQuoteBaseCite {
    // Output the placeholder if it hasn't been output already.
    @if (not index($_o-quote-placeholders-defined, 'o-quote-cite')) {
        %_o-quote-cite {

        }
        $_o-quote-placeholders-defined: append($_o-quote-placeholders-defined, 'o-quote-cite');
    }

    // Extend the o-quote placeholder.
    @extend %_o-quote-cite;
}
```

What do you think?

Reply:
yeah, that seems good. i'll do it that way.